src/cargo/ops/tree/format/mod.rs

```diff
@@ -97,6 +101,16 @@ impl<'a> fmt::Display for Display<'a> {
                         Chunk::Features => {
                             write!(fmt, "{}", features.join(","))?;
                         }
+                        Chunk::LibName => {
+                            if let Some(target) = package
+                                .manifest()
+                                .targets()
+                                .iter()
+                                .find(|target| target.is_lib())
+                            {
+                                write!(fmt, "{}", target.name())?;
```

Comment:
Would it make sense to use `crate_name()` here instead?  That way, it will translate dashes to underscores which I think may be more in spirit of what you're after?

Reply:
I like that behavior a lot better, personally.